pages/index.tsx

```diff
@@ -12,6 +13,14 @@ import useTranslation from 'next-translate/useTranslation'
 const Home: NextPage = () => {
   const isMounted = useMounted()
   const { t } = useTranslation('common')
+  const { address, isConnected } = useAccount()
+  const henkakuV2 = process.env
+    .NEXT_PUBLIC_CONTRACT_HENKAKUV2_ADDRESS as `0x${string}`
+  const nengajo = process.env
+    .NEXT_PUBLIC_CONTRACT_NENGAJO_ADDRESS as `0x${string}`
+  const { approved, allowanceValue } = useApproval(henkakuV2, nengajo, address)
```

Comment:
`allowanceValue` is not used, so there is no need to caoture its value. I guess `{ approved }` should be enough.

The comment on the following line is not needed either.

Reply:
Fixed.